feature-libs/order/core/connectors/replenishment-order.adapter.ts

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
I know they were there ini @spartacus/core, but it's odd to see adapters here instead of in occ/adapters.  Should wee refactor this in a followup ticket?

Reply:
We put the `abstract` adapter with the connector together in all places.